dev-java/byte-buddy/files/byte-buddy-1.14.14-SkipTests.patch

```diff
@@ -0,0 +1,351 @@
+There were 47 failures:
+1) testNoParameterNameAndModifiers(net.bytebuddy.pool.TypePoolDefaultMethodDescriptionTest)
+java.lang.AssertionError: 
+Expected: is <true>
+     but: was <false>
+	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
+	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
+	at net.bytebuddy.description.method.AbstractMethodDescriptionTest.testNoParameterNameAndModifiers(AbstractMethodDescriptionTest.java:372)
+2) testDebuggingSymbols(net.bytebuddy.asm.AdviceLocalVariableArrayTest)
+java.lang.AssertionError: 
+Expected: a string containing "\"b\""
```

Comment:
this error is related to the fact that we don't compile the class with debug symbols so we get a different variable name (`local8` instead of `b`) and hence the test fails